examples/esp32/xively_demo/main/main.c

```diff
@@ -289,7 +287,7 @@ int8_t app_fetch_user_config( user_data_t* dst )
 
     /* Wait for a button press for a few seconds while flashing the LED. If the button is
      * pressed, clear the contents of user_config to force the provisioning process */
-    for ( int i = 40; i > 0; i-- )
+    for ( int i = 39; i >= 0; i-- )
```

Comment:
What's with the magic number here?

Reply:
Can it be defined at the top as a constant / Define?  
Or is there a platform system way to correlate ticks to time?

Reply:
Oh, and add to a ReadMe file somewhere in the project that you're concerned that the device reboots itself when the xively process crashes, and that people should think of another solution for their product . Then direct them to the code in the file for more information.

Reply:
If the comment is only in the code then it's likely to remain hidden from users.